wremnants/plot_tools.py

```diff
@@ -50,12 +49,12 @@ def addLegend(ax, ncols=2, extra_text=None):
     ax.legend(handles=handles, labels=labels, prop={'size' : 20*(0.7 if shape == 1 else 1.3)}, ncol=ncols, loc='upper right')
 
 def makeStackPlotWithRatio(histInfo, stackedProcs, label="nominal", unstacked=None, xlabel="", ylabel="Events/bin", 
-                rrange=[0.9, 1.1], ymax=None, xlim=None, binwnorm=None, select={}, nlegcols=2, action=None, extra_text=None):
+                rrange=[0.9, 1.1], ymax=None, xlim=None, binwnorm=None, select={}, action=None, extra_text=None):
```

Comment:
I think a few more of these changes are intentional, better to double check it

Reply:
indeed, I accidentally deleted the nlegcols argument in 2 functions, where they should be kept. reverted the change.